### ISSUE TITLE
GMA v3.6.0: Out-of-area grid zoom/load [rt.marine-geo.org #9094]

### ISSUE THREAD

**Issue by nevilleshane** [iedadata/geomapapp#534]
Migrated from [rt.marine-geo.org#9094](http://rt.marine-geo.org/Ticket/Display.html?id=9094) (status was 'resolved')

Requestors:
* andrewg@ldeo.columbia.edu

Attachments:
* [hbbjdgje.png](http://rt.marine-geo.org/Ticket/Attachment/120121/63703/hbbjdgje.png)
* [hdeifhhb.png](http://rt.marine-geo.org/Ticket/Attachment/120121/63704/hdeifhhb.png)

From andrewg@ldeo.columbia.edu on 2015-10-14 17:43:32
:
```
This new item is closely related to closed ticket 
http://rt.marine-geo.org/Ticket/Display.html?id=6571 but is being 
treated as a new ticket.

In the case where a user zooms to an area without loading any layers, 
then loads an out-of-area contributed grid, only the top portion of the 
grid window comes up - there is no message about the grid being 
out-of-area. Then, when the user clicks the zoom button in the Layer 
Manager, the grid does not load automatically. The grid is loaded only 
after the user clicks Reload.

Example: Open GMA and zoom in to California. Next, open Focus Sites > 
R2k > Mid-Atlantic Ridge > Bathymetry Grids > Lucky Strike region > 
Bathymetry (10m) KN145-19. The Contributed Grids window opens but does 
not display the "Grid falls out of area" message:


When the Layer Manager zoom button is clicked, , the map window 
correctly jumps to the MAR region but does not automatically load the 
grid. Only after the user clicks the Contributed Grids window button 
will the grid show up.

Andrew



```

From nshane@ldeo.columbia.edu on 2020-02-24 15:28:27
:
```
Ticket migrated to github as https://github.com/iedadata/geomapapp/issues/135
```
